### PR TITLE
More Beeper clean up

### PR DESCRIPTION
**Pull request by AnHardt**
Corrected wrong order of parameters.

### CODE REVIEW

**Comment by AnHardt** [comment]
More Beeper clean up.

Centralised beep code to lcd_buzz().
From gcode_M300() and lcd_quick_feedback().

Enforced max. duration for M300 to 5 seconds.

Corrected description in 'Configuration.h'

LCD_FEEDBACK_FREQUENCY_HZ 0 gives now a silent delay of
LCD_FEEDBACK_FREQUENCY_DURATION_MS.

File: Marlin/Configuration.h [hunk 1]
```diff
@@ -681,7 +680,7 @@ const bool Z_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the logic
 
 // The RepRapDiscount Smart Controller (white PCB)
 // http://reprap.org/wiki/RepRapDiscount_Smart_Controller
-//#define REPRAP_DISCOUNT_SMART_CONTROLLER
+#define REPRAP_DISCOUNT_SMART_CONTROLLER
```
Comment (tonnico) [inline Marlin/Configuration.h]: you forgot to comment that again

**Comment by thinkyhead** [comment]
If the LCD click still sounds nice then I like this update. Is `lcd.buzz()` a new thing, or was it just overlooked in earlier versions of Marlin?

File: Marlin/Marlin_main.cpp [hunk 2]
```diff
@@ -3965,20 +3965,8 @@ inline void gcode_M226() {
   inline void gcode_M300() {
     uint16_t beepS = code_seen('S') ? code_value_short() : 110;
     uint32_t beepP = code_seen('P') ? code_value_long() : 1000;
-    if (beepS > 0) {
-      #if BEEPER > 0
-        tone(BEEPER, beepS);
-        delay(beepP);
-        noTone(BEEPER);
-      #elif defined(ULTRALCD)
-        lcd_buzz(beepS, beepP);
```
Comment (thinkyhead) [inline Marlin/Marlin_main.cpp:3974]: good catch

**Pull request merged**